src/scrabble.js

```diff
@@ -1,6 +1,50 @@
+const letters = {
+    lettersA: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T', 'a', 'e', 'i', 'o', 'u', 'l', 'n', 'r', 's', 't'],
+    lettersB: ['D', 'G', 'd', 'g'],
+    lettersC: ['B', 'C', 'M', 'P', 'b', 'c', 'm', 'p'],
+    lettersD: ['F', 'H', 'V', 'W', 'Y', 'f', 'h', 'v', 'w', 'y'],
+    lettersE: ['K', 'k'],
+    lettersF: ['J', 'X', 'j', 'x'],
+    lettersG: ['Q', 'Z', 'q', 'z']
+}
+
 class Scrabble {
-  // Write your implementation here
+    // Write your implementation here
+    constructor(word) {
+        this.word = word
+    }
 
```

Comment:
in here you could then handle lower case words like:
`let word = this.word.trim().toUpperCase();`